docs/installation.md

```diff
@@ -54,7 +54,7 @@ A [GitHub Actions](https://docs.github.com/en/actions) workflow is an automated
     ```{literalinclude} ../sample-workflow.yml
     :language: yaml
     ```
-3. **Edit the value of the `INVENIO_SERVER` variable near the top of the file above** ↑
+3. **Edit the value of the `INVENIO_SERVER` variable near the top of the file above** ↑. For CaltechDATA the value should be https://data.caltech.edu
```

Comment:
Can you please put the URL in literal/fixed-with font? Doing so will also help with another issue: there should be a period at the end of the sentence, and while it _would_ be confusing to put a period at the end of the URL, setting off the URL in a different typeface ought to help make it clear that the sentence-ending period is not part of the URL.